clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -221,6 +221,16 @@ object GarbageCollector {
     val readTimeout = hc.get(LAKEFS_CONF_API_READ_TIMEOUT_SEC_KEY)
     val maxCommitIsoDatetime = hc.get(LAKEFS_CONF_DEBUG_GC_MAX_COMMIT_ISO_DATETIME_KEY, "")
     val runIDToReproduce = hc.get(LAKEFS_CONF_DEBUG_GC_REPRODUCE_RUN_ID_KEY, "")
+
+    val markId = hc.get(LAKEFS_CONF_GC_MARK_ID, UUID.randomUUID().toString)
```

Comment:
Nit: Can we please pick something other than a UUID?  UUIDs are long, hard to read out (because hex), support multiple bad versions and only one good version, and are pretty ugly.  Nanoid maybe?

Reply:
sure, let's do that in a PR down the road